### PR TITLE
Don't expose URNs for anon orgs on v1 broadcasts endpoint

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/api/v1/serializers.py [hunk 1]
```diff
@@ -1467,7 +1467,10 @@ class BroadcastReadSerializer(ReadSerializer):
     status = serializers.ReadOnlyField()
 
     def get_urns(self, obj):
-        return [urn.urn for urn in obj.urns.all()]
+        if obj.org.is_anon:
+            return []
```
Comment (rowanseymour) [inline temba/api/v1/serializers.py]: `[]` vs `None`. Discuss.

Comment (nicpottier) [inline temba/api/v1/serializers.py]: `[]` is easier for clients to deal with, `None` feels more semantically correct. (implies more we are hiding something than it is an empty list, at least to me)

Probably prefer the latter? Anybody else?

**Comment by rowanseymour** [comment]
Updated to use `None` instead of `[]`

**Comment by nicpottier** [comment]
Yarp

**Pull request merged**